linux_os/guide/services/ntp/package_chrony_installed/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: fedora,ocp4,ol8,rhel8
+
+title: 'The Chrony package is enabled'
+
+description: |-
+    System time should be synchronized between all systems in an environment. This is
+    typically done by establishing an authoritative time server or set of servers and having all
+    systems synchronize their clocks to them.
+    You can install the package with the following command:
+    <pre># dnf install chrony</pre>
```

Comment:
We use Jinja macros that substitute the right command (dnf/yum/apt/...). 
Try this: `{{{ describe_package_install(package="chrony") }}}`